tests/integration/dashboard.spec.ts

```diff
@@ -0,0 +1,10 @@
+import {test, expect} from "../admin-pages";
+
+test.describe('Dashboard', async () => {
+  test('loads page', async ({ dashboardPage }) => {
+    await dashboardPage.goto();
+    await expect(dashboardPage.getActiveNavItem).toContainText('Dashboard')
+    await expect(dashboardPage.getQueriesOverTime).toBeVisible();
+    await expect(dashboardPage.getQueryTypesPie).toBeVisible();
+  })
+});
```

Comment:
1 of 2 actual tests in this PR - just access the dashboard and make sure things are visible.